### PR TITLE
Qa 2202 set operations automation ids

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Automation IDs for set operation selection screen and main app
- Automation IDs for set selection modal

### CODE REVIEW

File: packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx [hunk 1]
```diff
@@ -100,17 +100,22 @@ const SaveSelectionAsSetModal: React.FC<SaveSelectionAsSetModalProps> = ({
...
           required
           label="Save top:"
           min={1}
           max={max}
           {...form.getInputProps("top")}
         />
-        <p className="text-sm pb-2 pt-1">
+        <p
+          data-testid="text-up-to-top-items-can-be-saved"
+          className="text-sm pb-2 pt-1"
```
Comment (uc-pperumala) [inline packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx:112]: @bpierce01 are these your changes? to the className?

**Review by uc-pperumala — APPROVED** [review]
Just one comment related to changes for the className. Otherwise, everything else looks good. Approved.

File: packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx (hunk 1 above)
Comment (bpierce01) [reply packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx:112]: The classname didn't change just the formatting bc of pre-commit

Comment (uc-pperumala) [reply packages/portal-proto/src/components/Modals/SetModals/SaveSelectionModal.tsx:112]: Sounds good! Thank you for the clarification!

File: packages/portal-proto/src/components/Modals/SetModals/SetTable.tsx [hunk 2]
```diff
@@ -52,6 +52,7 @@ const SelectCell: React.FC<SelectCellProps> = ({
           />
         ) : (
           <Radio
+            data-testid={`radio-select-set`}
```
Comment (paribartandhakal) [inline packages/portal-proto/src/components/Modals/SetModals/SetTable.tsx:55]:
```suggestion
            data-testid="radio-select-set"
```
Nitpick.

File: packages/portal-proto/src/features/set-operations/SelectionPanel.tsx [hunk 3]
```diff
@@ -399,13 +403,15 @@ const SelectionPanel: React.FC<SelectionPanelProps> = ({
...
           <Link
+            data-testid="link-manage-sets"
             href="/manage_sets"
             className="text-utility-link font-content underline"
           >
             Manage Sets
           </Link>{" "}
           or in analysis tools (e.g.{" "}
           <Link
+            data-testid="link-mutation-frequnecy"
```
Comment (paribartandhakal) [inline packages/portal-proto/src/features/set-operations/SelectionPanel.tsx:414]: typo:
```suggestion
            data-testid="link-mutation-frequency"
```

**Review by paribartandhakal — APPROVED** [review]
Leaving two comments.

**Pull request merged**